### DISCUSSION TITLE
"Work" folder content

### DISCUSSION THREAD

**Discussion by Wanderlust1984** [flybywiresim/aircraft#4148] [Q&A] [answered]
Hi,

After the latest MFS2020 update i got my TCA Quadrant not calibrated again. Actually, following the same steps when I did it properly I realized that "Work" folder is empty and i can not edit the file ThrottleConfiguration.ini anymore .
So, I was wondering if I can copy somewhere the folder content with all this files. I can not find this folder here...
I really appreciate your help cause i quite desperated. This time I am not able to calibrate it again.
This folder is located in my case in LOCALSTATE/PACKAGES/flybywire-aircraft-a320-neo/work

Thank you

**Comment by SebSkarp** [comment]
The file you are looking for should be created after you run the sim with the mod active. Start any flight with a32nx and the file should appear after this. It's stated in the instructions if I'm not mistaken, and was also the case for me.

**Comment by Annolago** [comment]
Just copy the old one from the old folder to the new location

**Reply by Wanderlust1984** [reply]
I can't. I dont know why but the older one is empty.

**Reply by Annolago** [reply]
Just use notepad in windows to create a file called "ThrottleConfiguration.ini" and then copy the recommended settings from the warning page that is linked at the main A32NX page here on github
https://github.com/flybywiresim/a32nx/tree/autopilot/docs
That is all you need to do. Once you have created the file it will automatically be used by FS next time you start it.

**Comment by mojoair** [comment, marked as answer]
In order for the ThrottleConfiguration.ini to be created, you need to access the Calibration via the settings button in the EFB Pad.
Once you manipulate/change any of the value of the calibration on the EFB you must click apply and save. Once you do that, a ThrottleConfiguration.ini file will be generated in this new work folder.

You can try the following in the new throttle config file as a starting point. My settings are with a fully linear with 0 detent in MSFS settings.

[throttle_common]
reverse_on_axis = true

[throttle_axis_1]
reverse_low = -1.000000
reverse_high = -0.980000
reverse_idle_low = -0.720000
reverse_idle_high = -0.620000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.060000
flex_mct_low = 0.410000
flex_mct_high = 0.590000
toga_low = 0.970000
toga_high = 1.000000

[throttle_axis_2]
reverse_low = -1.000000
reverse_high = -0.980000
reverse_idle_low = -0.720000
reverse_idle_high = -0.620000
idle_low = -0.550000
idle_high = -0.470000
climb_low = -0.020000
climb_high = 0.060000
flex_mct_low = 0.410000
flex_mct_high = 0.590000
toga_low = 0.970000
toga_high = 1.000000

**Reply by Wanderlust1984** [reply]
I never touch the EFB, shall I run a flight at wherever place and try to modify the calibration through the EFB? Actually, I did not know that I could use the EFB. 
I will try it anyway. Thanks a lot.
UPDATE: I was able to calibrate de EFB and the file it was created again. Thanks¡

**Reply by Annolago** [reply]
Mojoair

Could you please explain why you have high and low settings? Never seen that before and how do you switch between them - or do you?

**Comment by MapleNAV-TCASFAULT** [comment]
https://www.youtube.com/watch?v=OCCJOQyWyq8&t=849s&ab_channel=320SimPilot You can watch 320 simpilot's video on how to set up your throttle quadrant through the efb
